x-pack/plugins/security_solution/public/common/utils/alerts.ts

```diff
@@ -124,7 +124,7 @@ export interface Alert {
 
 // generates default grouping option for alerts table
 export const getDefaultGroupingOptions = (tableId: TableId): GroupOption[] => {
-  if (tableId === TableId.alertsOnAlertsPage) {
+  if (tableId === TableId.alertsOnAlertsPage || tableId === TableId.alertsRiskInputs) {
```

Comment:
I'm not sure if there was a reason why grouping was limited to only the alerts page versus the rule-details page as well, but would be good to confirm it makes sense with either @stephmilovic or @YulNaumenko